WordPress/Classes/ViewRelated/Tools/Views/TimeZoneSearchBar.swift

```diff
@@ -0,0 +1,62 @@
+import UIKit
+
+final class TimeZoneSearchBar: UIView {
+
+    @IBOutlet private weak var searchWrapperView: SearchWrapperView!
+
+    @IBOutlet private weak var searchWrapperViewHeightConstraint: NSLayoutConstraint!
+
+    @IBOutlet private weak var suggestionLabel: UILabel!
+
+    @IBOutlet private weak var suggestionButton: UIButton!
+
+    /// Callback called when the button is tapped
+    var tapped: (() -> Void)?
+
+    required init?(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)
+    }
+
+    class func makeFromNib() -> TimeZoneSearchBar {
+        return Bundle.main.loadNibNamed(Constants.nibIdentifier,
+                                        owner: self,
+                                        options: nil)?.first as! TimeZoneSearchBar
+    }
+
+    // MARK: - Convenience Initializers
+
+    override func awakeFromNib() {
+        super.awakeFromNib()
+
+        suggestionLabel.text = Localization.suggestion
+        suggestionButton.addTarget(self, action: #selector(buttonTapped), for: .touchUpInside)
+    }
+
+    @objc private func buttonTapped() {
+        tapped?()
+    }
+
+    func configureSearchWrapperView(searchBar: UISearchBar) {
```

Comment:
I wonder if we should make the search bar mandatory? You can still create an instance of this type and forget to call this method, which will create it without a search bar?
Any suggestion on how to refactor this type?

Reply:
Sounds good!  I've refactored so that the `searchBar` and default `TimeZoneIdentifier` which will be the suggestion is required to be passed into the factory method (`makeFromNib(searchBar: UISearchBar, timezone: String)`)